src/main/java/frc/robot/subsystems/scoring/ScoringSubsystem.java

```diff
@@ -328,6 +330,13 @@ private void shoot() {
     private void ampShoot() {
         shooterIo.setKickerVolts(-12); // TODO: Test if this kicks note forward or backward
 
+        // Revert to amp prime after we shoot the note
+        // This will keep the arm up and allow the driver to drive away from the amp before the arm
+        // comes down
```

Comment:
Question: the concept of operations is that the driver shoots the note right?

Reply:
Driver or masher can shoot, they both have shoot buttons (Driver: Right joystick, button 4, masher: right bumper). Either one can shoot, depending on the preferences of the drive team. We still need to keep the arm up because it's impossible to drive away in time while the arm is controlling down to the intake angle at full speed without smashing the arm against the wall.

Reply:
Can't really change code for this one but if you're satisfied with this answer you can resolve the convo, I've fixed the other two issues so it's ready for review again.

Reply:
We can discuss at shop more, but as long as the driver is able to shoot I think I'm good.

My main concern is how much workload is there on the masher to hold this arm up. It sounds like not much but there will be some communication needed

Reply:
yes this is a fair concern. Also, driver will probably be better at reacting to when robot is correctly parked so if they can go ahead and spit into the amp that would be better. We should instruct masher to put robot into amp mode (D-Pad right) as soon as we decide we're going to score in the amp. Then, they should start warming up immediately after we intake the note so that everything is done well in time for the driver to get the robot to the amp.